ws-services/src/main/java/org/egov/waterconnection/constants/WCConstants.java

```diff
@@ -95,57 +94,72 @@ private WCConstants() {
+
 	public static final List<String> NOTIFICATION_ENABLE_FOR_STATUS = Collections
 			.unmodifiableList(Arrays.asList(INITIATE_INITIATED, REJECT_REJECTED,
 					SEND_BACK_TO_CITIZEN_PENDING_FOR_CITIZEN_ACTION, SEND_BACK_FOR_DO_PENDING_FOR_DOCUMENT_VERIFICATION,
 					SEND_BACK_PENDING_FOR_FIELD_INSPECTION, VERIFY_AND_FORWORD_PENDING_FOR_FIELD_INSPECTION,
 					VERIFY_AND_FORWARD_PENDING_APPROVAL_FOR_CONNECTION, APPROVE_FOR_CONNECTION_PENDING_FOR_PAYMENT,
 					PAY_PENDING_FOR_CONNECTION_ACTIVATION, ACTIVATE_CONNECTION_CONNECTION_ACTIVATED,
-					EDIT_PENDING_FOR_DOCUMENT_VERIFICATION, EDIT_PENDING_FOR_FIELD_INSPECTION));
-	
-	public static final String  USREVENTS_EVENT_TYPE = "SYSTEMGENERATED";
-	
-	public static final String  USREVENTS_EVENT_NAME = "WATER CONNECTION";
-	
-	public static final String  USREVENTS_EVENT_POSTEDBY = "SYSTEM-WS";
-	
+					EDIT_PENDING_FOR_DOCUMENT_VERIFICATION, EDIT_PENDING_FOR_FIELD_INSPECTION,
+					SUBMIT_APPLICATION_STATUS_CODE, APPROVE_CONNECTION_STATUS_CODE));
+
+	public static final String USREVENTS_EVENT_TYPE = "SYSTEMGENERATED";
+
+	public static final String USREVENTS_EVENT_NAME = "WATER CONNECTION";
+
+	public static final String USREVENTS_EVENT_POSTEDBY = "SYSTEM-WS";
+
 	public static final String VARIABLE_WFDOCUMENTS = "documents";
 
-    public static final String VARIABLE_PLUMBER = "plumberInfo";
-    
-    public static final String WC_ROADTYPE_MASTER = "RoadType";
-	
-	public static final List<String> FIELDS_TO_CHECK = Collections
-			.unmodifiableList(Arrays.asList("rainWaterHarvesting", "waterSource", "meterId", "meterInstallationDate",
-					"proposedPipeSize", "proposedTaps", "pipeSize", "noOfTaps", "oldConnectionNo", "roadType",
-					"roadCuttingArea", "connectionExecutionDate", "connectionCategory", "connectionType",
-					"documentType", "fileStoreId", "licenseNo"));
+	public static final String VARIABLE_PLUMBER = "plumberInfo";
+
+	public static final String WC_ROADTYPE_MASTER = "RoadType";
+
+	public static final String PROPERTY_OWNERTYPE = "OwnerType";
+
+	public static final String PROPERTY_MASTER_MODULE = "PropertyTax";
+
+	public static final List<String> FIELDS_TO_CHECK = Collections.unmodifiableList(Arrays.asList("rainWaterHarvesting",
+			"waterSource", "meterId", "meterInstallationDate", "proposedPipeSize", "proposedTaps", "pipeSize",
+			"noOfTaps", "oldConnectionNo", "roadType", "roadCuttingArea", "connectionExecutionDate",
+			"connectionCategory", "connectionType", "documentType", "fileStoreId", "licenseNo"));
 
 	public static final String WS_EDIT_SMS = "WS_EDIT_SMS_MESSAGE";
-	
+
 	public static final String WS_EDIT_IN_APP = "WS_EDIT_IN_APP_MESSAGE";
+
+	public static final String DEFAULT_OBJECT_EDIT_SMS_MSG = "Dear <Owner Name>, Your Application <Application number>  for a New <Service> Connection has been edited. For more details, please log in to <mseva URL> or download <mseva app link>.";
```

Comment:
These messages needs to come from localization and not directly embedded in code

Reply:
made changes to get message from localisation